TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeyManager.swift

```diff
@@ -21,7 +21,7 @@ import Foundation
 
 ///:nodoc:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/c51c4513007d818dab4b64cd25c837959c7ec03d/TwilioVerifySDK/TwilioSecurity/Sources/Keychain/KeyManager.swift#L22"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>KeyManager.swift:22</code></td>
    </tr>
  </tbody>
</table>